src/create-client.ts

```diff
@@ -328,5 +333,6 @@ An authorization_code was supplied for a login which did not originate at the ap
     getUser,
     getAccessToken,
     signOut,
+    dispose,
```

Comment:
This is the only public API change. I think it makes sense to have, though if anyone has concerns, I can probably find a way to keep it private for the moment.